ui/tabs.js

```diff
@@ -621,7 +621,7 @@ return $.widget( "ui.tabs", {
 		}
 
 		function show() {
-			eventData.newTab.closest( "li" ).addClass( "ui-tabs-active ui-state-active" );
+			that._addClass( eventData.newTab.closest( "li" ), "ui-tabs-active", "ui-state-active" );
```

Comment:
Nevermind. I saw that the `oldTab` case was used twice, but they're in exclusive blocks.